.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -136,8 +136,12 @@ jobs:
               mv target/docker-sonic-vs.gz target/docker-sonic-vs-asan.gz
             fi
             if [ "$(K8S_OPTIONS)" == 'INCLUDE_KUBERNETES_MASTER=y' ]; then
-              make $BUILD_OPTIONS $(K8S_OPTIONS) target/sonic-vs.img.gz
-              mv target/sonic-vs.img.gz target/sonic-vs-k8s.img.gz
+              make $BUILD_OPTIONS INCLUDE_KUBERNETES_MASTER=y target/sonic-vs.img.gz
+              gzip -kd target/sonic-vs.img.gz
```

Comment:
Looks like the keep option "-k" can be removed.

Reply:
> Looks like the keep option "-k" can be removed.

Yes, you are correct. Thanks for this comment, fixed.